logger.js

```diff
@@ -46,13 +47,20 @@ function pinoLogger (opts, stream) {
     }, 'request completed')
   }
 
+  function onReqAborted() {
+    this.res.statusCode = 0;
```

Comment:
@marcbachmann, if we don't do it there, then the log will be with 200 statusCode, that's why it's here